<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>DeveloperInfo</class>
    <widget class="QDialog" name="DeveloperInfo">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>280</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Developer Info</string>
        </property>
        <property name="styleSheet">
            <string notr="true">
                background-color: #e0f7f1;
                color: #4a148c;
                font-family: "Segoe UI";
                font-size: 14px;
            </string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="titleLabel">
                    <property name="text">
                        <string>Developer Information</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">
                            font-size: 18px;
                            font-weight: bold;
                            color: #2e7d32;
                        </string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLabel" name="emailLabel">
                    <property name="text">
                        <string>
                            <![CDATA[
    <span style="white-space:nowrap;">
      <img src="images/email.png" width="20" height="20" style="vertical-align:middle; margin-right:6px;" />
      Peyman.ghamari.g.p@gmail.com
    </span>
   ]]>
                        </string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>


            </item>
            <item>
                <widget class="QLabel" name="githubLabel">
                    <property name="text">
                        <string>
                            <![CDATA[
    <span style="white-space:nowrap;">
      <img src="images/github.png" width="20" height="20" style="vertical-align:middle; margin-right:6px;" />
      <a href="https://github.com/Peyman-Ghamari">Peyman-Ghamari</a>
    </span>
   ]]>
                        </string>
                    </property>
                    <property name="openExternalLinks">
                        <bool>true</bool>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>


            </item>
            <item>
                <widget class="QLabel" name="telegramLabel">
                    <property name="text">
                        <string>
                            <![CDATA[
    <span style="white-space:nowrap;">
      <img src="images/telegram.png" width="20" height="20" style="vertical-align:middle; margin-right:6px;" />
      <a href="https://t.me/p_ghamari">p_ghamari</a>
    </span>
   ]]>
                        </string>
                    </property>
                    <property name="openExternalLinks">
                        <bool>true</bool>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>

            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QLabel" name="copyrightLabel">
                    <property name="text">
                        <string>Â© Copyright by PM (Peyman Ghamari)</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">
                            color: #000000;
                            font-size: 12px;
                            font-style: italic;
                            background-color: #f3e5f5;
                            padding: 4px;
                            border-radius: 4px;
                        </string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
